README.md

```diff
@@ -17,3 +17,45 @@ Fork this repository, and submit your pull request providing documentation to th
 
 ### Product Specification
 * See wireframe.png
+
+
+# Documentation
+
+### requirements
+- `yarn@1.17.3`
+- `node@10.16.3`
+
+### install project dependencies
+```
+yarn
```

Comment:
@alexander-daniel Why use `yarn` rather than `npm`?

Reply:
I started using `yarn` at a time when `npm` was having issues with child dependency resolution and conflicts. I have found it to be anecdotally faster, but I know that both work great and ultimately you would be fine using either. since `npm v 5` things have more or less been solved and I would feel comfortable using either. So in this case, to answer your question, I chose to use `yarn` out of familiarity and personal choice.